README.md

```diff
@@ -3,40 +3,40 @@
 
-Este repositorio contiene derivaciones de Nix, módulos de NixOS y Home Manager
-para integrar AutoFirma, DNIeRemote y el Configurador FNMT en NixOS y Home
-Manager.
+This repository provides a suite of tools needed to interact with Spain’s public administration,
+alongside NixOS and Home Manager modules for easy integration. These tools include:
 
+- **AutoFirma** for digitally signing documents  
+- **DNIeRemote** for using an NFC-based national ID with an Android device as an NFC reader  
+- **FNMT Configurator** for securely requesting the personal certificate from the Spanish Royal Mint (**Fábrica Nacional de Moneda y Timbre**)  
+
+## Usage Example
 
-## Ejemplo de uso
 ```console
 $ nix run --accept-flake-config github:nilp0inter/autofirma-nix#dnieremote
 ```
 
-## AutoFirma en NixOS y Home Manager
-
-Se proporciona un módulo de NixOS para habilitar AutoFirma en NixOS y otro para
-Home Manager. Sólo es necesario habilitar uno de ellos, dependiendo de si se
-quiere habilitar AutoFirma a nivel de sistema o de usuario.
+## AutoFirma on NixOS and Home Manager
 
-Una vez habilitado uno de los módulos, si se quiere utilizar AutoFirma en
-Firefox, es necesario ejecutar el comando `autofirma-setup` (ver más abajo).
+A NixOS module is provided to enable AutoFirma on NixOS and another one for Home Manager.
+You only need to enable one of them, depending on whether you want AutoFirma
+system-wide or at the user level.
 
+Once you have enabled one of these modules, if you want to use AutoFirma in Firefox,
+you must run the `autofirma-setup` command (see below).
 
-### Configuración de Home Manager
+### Home Manager Configuration
 
-La integración de AutoFirma en Home Manager habilita el comando `autofirma` para
-el firmado de documentos PDF y configura el navegador Firefox (si está habilitado
-mediante la opción `programs.firefox.enable`) para que utilice AutoFirma en
-sitios web que lo requieran.
+The integration of AutoFirma in Home Manager enables the `autofirma` command for
+signing PDF documents and configures the Firefox browser (if enabled through
+`programs.firefox.enable`) to use AutoFirma on websites that require it.
 
-Adicionalmente, se puede habilitar la integración con el DNIe y el DNIe por NFC
-desde un móvil Android usando DNIeRemote.
+Additionally, you can enable DNIe integration, including NFC-based DNIe from an
+Android mobile via DNIeRemote.
 
-`autofirma-nix` proporciona un módulo de Home Manager que debe ser importado en
-el fichero de configuración de Home Manager.  Dependiendo del tipo de
-instalación de Home Manager la configuración puede variar ligeramente.  A
-continuación se muestran ejemplos para una configuración de tipo standalone.
+`autofirma-nix` provides a Home Manager module that you must import in your Home
```

Comment:
Is *must* here too strong? We wonder if *can*, *could* or *should* are better alternative implementations ;-)

Co-authored-by: @CesarGallego